### PR TITLE
Add max chip amount

### LINKED ISSUE #723: Cap chips in inventory at 20

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Add visual feedback for the max cap

- Add chip sprite to bidsigns for consistency

### CODE REVIEW

**Review by toberge — CHANGES_REQUESTED** [review]
Seems to work as intended, though the color choice isn't ideal. It shouldn't be pure #ff0000 >:(, try using a color that is in use elsewhere in the game. And perhaps show 20 (MAX) instead of just MAX on the chip popup.

image

File: Assets/Scripts/UI/PlayerHUDController.cs [hunk 1]
```diff
@@ -207,8 +207,11 @@ public void OnChipChange(int amount)
         // TODO replace by not requiring UI stuff on the bidding players
         if (!chipAmount)
             return;
-
-        chipAmount.text = amount.ToString();
+        bool isMax = amount == PlayerIdentity.MaxChips;
+        Debug.Log(PlayerIdentity.MaxChips);
+        Debug.Log(amount);
```
Comment (toberge) [inline Assets/Scripts/UI/PlayerHUDController.cs]: Please remove these log statements :)))))

Comment (Fueredoriku) [reply Assets/Scripts/UI/PlayerHUDController.cs]: Fixed 👍

**Review by toberge — APPROVED** [review]
Looks like a better choice of red 👍

**Pull request merged**